libs/blocks/chart/chart.js

```diff
@@ -465,6 +465,10 @@ const init = async (el) => {
         ));
       })
       .catch((error) => console.log('Error loading script:', error));
+  } else {
+    fetchData(dataLink).then((data) => {
+      // TODO: MWPW-113009 Oversized Number Visualization
+    });
```

Comment:
Since we're not sure if/when we'll be getting to the oversized number, I think it's better we don't include this for now

Reply:
Makes sense, I just wanted to have an example for `fetchData(dataLink).then((data) => {});`